variants/deneyapkartg/pins_arduino.h

```diff
@@ -2,26 +2,35 @@
 #define Pins_Arduino_h
 
 #include <stdint.h>
+#include "soc/soc_caps.h"
+
+#define USB_VID            0x303A
+#define USB_PID            0x814A
+#define USB_MANUFACTURER   "Turkish Technnology Team Foundation (T3)"
+#define USB_PRODUCT        "DENEYAP KART G"
+#define USB_SERIAL         "" // Empty string for MAC adddress
 
 #define EXTERNAL_NUM_INTERRUPTS 22
 #define NUM_DIGITAL_PINS        22
 #define NUM_ANALOG_INPUTS       6
 
+static const uint8_t LED_BUILTIN = SOC_GPIO_PIN_COUNT+10;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+#define LED_BUILTIN LED_BUILTIN
+#define RGB_BUILTIN LED_BUILTIN
+#define RGBLED	LED_BUILTIN
+#define RGB_BRIGHTNESS 64
+
 #define analogInputToDigitalPin(p)  (((p)<NUM_ANALOG_INPUTS)?(analogChannelToDigitalPin(p)):-1)
 #define digitalPinToInterrupt(p)    (((p)<NUM_DIGITAL_PINS)?(p):-1)
 #define digitalPinHasPWM(p)         (p < EXTERNAL_NUM_INTERRUPTS)
 
-static const uint8_t RGBLED  = 10;
 static const uint8_t GPKEY  = 9;
-
-#define RGB_BUILTIN RGBLED
-#define RGB_BRIGHTNESS 64
-
 #define KEY_BUILTIN GPKEY
-#define BUILTIN_KEY KEY_BUILTIN
+#define BUILTIN_KEY GPKEY
 
-static const uint8_t TX = 20;
-static const uint8_t RX = 21;
+static const uint8_t TX = 21;
+static const uint8_t RX = 20;
```

Comment:
Are you sure the pins are right? In your docs in [this table ](https://cdn.deneyapkart.org/media/uploads/2022/12/20/dyg_detaylpinozellikleritablos.png) I see that:
```
TX = D1 = 21
RX = D0 = 20
```
but in pins_arduino.h the `D0 = 21` and `D1 = 20` which is opposite. Can you check it please?

Reply:
> Are you sure the pins are right? In your docs in [this table ](https://cdn.deneyapkart.org/media/uploads/2022/12/20/dyg_detaylpinozellikleritablos.png) I see that:
> 
> ```
> TX = D1 = 21
> RX = D0 = 20
> ```
> 
> but in pins_arduino.h the `D0 = 21` and `D1 = 20` which is opposite. Can you check it please?

Yes, I am sure. It was wrong that i fixed.